docs/configuration/local_php_overview.rst

```diff
@@ -0,0 +1,554 @@
+Local.php fields overview
+#########
+
+Most but not all Mautic configuration can be done from the user interface.
```

Comment:
This needs to be active rather than passive tense, and User Interface must always be capitalised.

```suggestion
Most but not all Mautic configuration is managed in the User Interface.```